grants_tagger_light/training/dataloaders/mesh_json_loader.py

```diff
@@ -0,0 +1,111 @@
+import json
+import numpy as np
+from transformers import AutoTokenizer
+from datasets import Dataset
+
+# TODO refactor the two load funcs into a class
+
+
+def _tokenize(batch, tokenizer: AutoTokenizer, x_col: str):
+    return tokenizer(
+        batch[x_col],
+        padding="max_length",
+        truncation=True,
+        max_length=512,
+    )
+
+
+def _label_encode(batch, mesh_terms_column: str, label2id: dict):
+    batch_labels = []
+    for sample_tags in batch[mesh_terms_column]:
```

Comment:
I think `batch_tags` is cleaner, you are not sampling

Reply:
rename `sample_tags` to `batch_tags`

Reply:
`example_tags` could be cleaner but up to you

Reply:
Fair. Renamed to `example_tags`